Library/Formula/dfmt.rb

```diff
@@ -13,6 +13,12 @@ class Dfmt < Formula
     sha256 "d306278e22b46e60b6474593e1107db753c4fefc907e66bd1f348cc29af0fc95" => :mavericks
   end
 
+  devel do
+    url "https://github.com/Hackerpilot/dfmt.git",
+        :tag => "v0.5.0-beta3",
+        :revision => "845358bb61603031b0817aed03097064c8f2553f"
+  end
```

Comment:
Apologies but you may have to set an explicit `version` here as well to include the `beta` tag. It's being filtered out at the moment:
```
dfmt: stable 0.4.4 (bottled), devel 0.5.0, HEAD
```